src/page-multi-facility/FacilityRowRtValuePill.tsx

```diff
@@ -1,61 +1,112 @@
+    text: string;
+    border: string;
   };
 } = {
-  [RateOfSpreadType.CONTROLLED]: {
+  [RateOfSpreadType.Controlled]: {
     text: Colors.green,
     border: Colors.teal,
   },
-  [RateOfSpreadType.INFECTIOUS]: {
+  [RateOfSpreadType.Infectious]: {
     text: Colors.darkRed,
     border: Colors.orange,
   },
-  [RateOfSpreadType.MISSING]: {
+  [RateOfSpreadType.Missing]: {
     text: Colors.forest,
     border: Colors.forest30,
   },
 };
 
-const RtValuePill = styled.div<{ spreadType: string }>`
-  .pill-circle {
+const RtValuePill = styled.div<{ spreadType: RateOfSpreadType }>`
+  ${PillCircle} {
     border-color: ${(props) => pillColors[props.spreadType].border};
     color: ${(props) => pillColors[props.spreadType].text};
   }
+
+  &:hover {
+    ${ChartTooltip} {
+      display: block;
+    }
+  }
+`;
+
+const PillTooltip = styled.div`
+  position: relative;
+
+  ${ChartTooltip} {
+    display: none;
+    max-width: 260px;
+    position: absolute;
+    right: 0px;
+    transform: translateY(calc(-100% - 4px)) translateX(2px);
+
+    &:after {
+      left: calc(100% - 21px);
+    }
+  }
+
+  ${TooltipLabel} {
+    white-space: pre-wrap;
+  }
 `;
 
 interface Props {
-  latestRt: number | null;
+  latestRt: number | null | undefined;
 }
 
-const rtSpreadType = (latestRt: number | null) => {
-  if (!latestRt) {
-    return RateOfSpreadType.MISSING;
+const rtSpreadType = (latestRt: number | null | undefined) => {
+  if (latestRt === null || latestRt === undefined) {
+    return RateOfSpreadType.Missing;
   } else if (latestRt > 1) {
-    return RateOfSpreadType.INFECTIOUS;
+    return RateOfSpreadType.Infectious;
   } else {
-    return RateOfSpreadType.CONTROLLED;
+    return RateOfSpreadType.Controlled;
   }
 };
 
-const displayRtValue = (latestRt: number | null) => {
-  return !latestRt ? "?" : latestRt;
+const displayRtValue = (latestRt: number | null | undefined) => {
+  return numeral(latestRt).format("0.0") || "?";
 };
 
 const FacilityRowRtValuePill: React.FC<Props> = ({ latestRt: latestRt }) => {
+  const hasDataTitle = `Rate of spread (Rt): ${displayRtValue(latestRt)}`;
+  const needsDataTitle = `Not enough data`;
+  const hasDataBody = `Numbers above 1 indicate how quickly the virus is spreading. If the value is below 1, the virus is on track to be extinguished at this facility.`;
+  const needsDataBody = `To calculate the rate of transmission, enter more than one day of facility case data.`;
+
   return (
-    <RtValuePill spreadType={rtSpreadType(latestRt)}>
-      <PillCircle className="pill-circle">
-        {displayRtValue(latestRt)}
-      </PillCircle>
-    </RtValuePill>
+    <>
+      <RtValuePill spreadType={rtSpreadType(latestRt)}>
+        <PillTooltip>
+          <ChartTooltip>
+            <TooltipContents>
+              <TooltipValue>
+                {!!latestRt ? hasDataTitle : needsDataTitle}
+              </TooltipValue>
+              <TooltipLabel>
+                {!!latestRt ? hasDataBody : needsDataBody}
```

Comment:
I did not flag this in the previous review, but this will also do the wrong thing if `latestRt` is zero

Reply:
Make fixing updates.